modules/tensor_mechanics/doc/content/source/materials/ComputeCrackedStress.md

```diff
@@ -84,10 +84,11 @@ where $\boldsymbol{\mathcal{M}}$ is the Jacobian_mult that was calculated by the
 ## Evolution Equation and History Variable
 
 To avoid crack healing, a history variable $H$ is defined that is the maximum energy density over the
-time interval $t=[0,t_0]$, where $t_0$ is the current time step, i.e.
+time interval $t=[0,t_0]$, where $t$ is the time and $t_0$ is the current time step, i.e.
 \begin{equation}
 H = \max_t (\Psi^{+})
 \end{equation}
+By default, the model is applied under the assumption that the material is undamaged at $t = 0$ (the history variable is initialized at zero, regardless of the initial condition on $c$). However, options are available to stabilize pre-existing cracks. When pre-existing cracks are desired, a threshold is applied to the initial condition on $c$ to determine the initial state of the history variable. These options can be used to influence where cracks nucleate and grow.
```

Comment:
It would be good if you can add the formulation that computes the initial history variable.